en/08_Changelogs/6.0.0.md

```diff
@@ -606,14 +606,25 @@ See the [full list of removed and changed API](#api-removed-and-changed) to see
 
 ### Changes to `LeftAndMain` and its subclasses {#leftandmain-refactor}
 
-[`LeftAndMain`](api:SilverStripe\Admin\LeftAndMain) has historically been the superclass for all controllers routed in `/admin/*` (i.e. all controllers used in the CMS). That class includes a lot of boilerplate functionality for setting up a menu, edit form, etc which a lot of controllers don't need.
+[`LeftAndMain`](api:SilverStripe\Admin\LeftAndMain) has historically been the superclass for all controllers routed in `/admin/*` (i.e. all controllers used in the CMS). It's also the superclass for admin-routed controllers which manage modal forms. That class includes a lot of boilerplate functionality for setting up a menu, edit form, etc which a lot of controllers don't need.
 
 A new [`AdminController`](api:SilverStripe\Admin\AdminController) has been created which provides the `/admin/*` routing functionality and permission checks that `LeftAndMain` used to be responsible for. If you have a controller which needs to be routed through `/admin/*` with the relevant CMS permission checks, but which does *not* need a menu item on the left or an edit form, you should update that class to be a subclass of `AdminController` instead.
 
-As a result of this change, to following classes are now direct subclasses of `AdminController`:
+The new [`FormSchemaController`](api:SilverStripe\Admin\FormSchemaController) class (which is a subclass of `AdminController`) now owns the logic required for injecting and managing forms inside modals.
 
-- [`SudoModeController`](api:SilverStripe\Admin\SudoModeController) - used to be a subclass of `LeftAndMain`.
-- [`CMSExternalLinksController`](api:SilverStripe\ExternalLinks\Controllers\CMSExternalLinksController) - used to be a direct subclass of [`Controller`](api:SilverStripe\Control\Controller).
+As a result of these changes, the following classes have had their class hierarchy updated:
+
+|class|old superclass|new superclass|
```

Comment:
```suggestion
|Class|Old super class|New super class|
```

Reply:
Would parent class be better than super class?

Reply:
The capitalisation isn't consistent with other tables in this page. Can you please look at the page in its rendered form (so you can see in context) and decide if you want the headers for all tables capitalised or lower-case? If you want them capitalised I'll update all the other tables as well.

> Would parent class be better than super class?

They mean the same thing so I don't mind either way.

Reply:
They should all be capitalized, looks wrong if it's lower case. Leave as super class if it's consistent

Reply:
I think it looks weird capitalized lol but I'll make the change.

Reply:
Done